index.html

```diff
@@ -10,6 +11,14 @@
     <body>
         <h1>CTL Event testing</h1>
         <div class="pagination-holder"></div>
+        <div class="search-wrapper">
+            <form role="search">
+                <input id="q" type="text" required="" class="search-box" placeholder="I'm searching for...">
+                <button class="close-icon" id="clear-search" type="reset"></button>
+                <button type="submit" id="search" style="display:none;"></button>
```

Comment:
We should add some text to these buttons - e.g. `<button>Search</button>`, so it doesn't look like this:
![2017-02-27-165531_231x38_scrot](https://cloud.githubusercontent.com/assets/59292/23381762/aa2b4c54-fd0d-11e6-883d-5bb13b4b687b.png)